string.c

```diff
@@ -11525,6 +11525,260 @@ rb_enc_interned_str_cstr(const char *ptr, rb_encoding *enc)
+ *                and all downcase characters upcased.
+ *
+ *  _Deletion_
+ *
+ *  - #delete:: Returns a copy of +self+ with characters removed
+ *  - #delete_prefix:: Returns a copy of +self+ with a given prefix removed.
+ *  - #delete_suffix:: Returns a copy of +self+ with a given suffix removed.
+ *  - #lstrip:: Returns a copy of +self+ with leading whitespace removed.
+ *  - #rstrip:: Returns a copy of +self+ with trailing whitespace removed.
+ *  - #strip:: Returns a copy of +self+ with leading and trailing whitespace removed.
+ *  - #[]=:: Returns a substring o +self+ determined by given index, range, regexp, or string.
+ *  - #chomp:: Returns a copy of +self+ with a trailing record separator removed, if found.
+ *  - #chop:: Returns a copy of +self+ with trailing whitespace or the last character removed.
+ *  - #squeeze:: Returns a copy of +self+ with contiguous duplicate characters removed.
+ *  - #[], #slice:: Returns a substring determined by a given index, start/length, or range, or string.
+ *  - #byteslice:: Returns a substring determined by a given index, start/length, or range.
+ *  - #chr:: Returns the first character.
+ *
+ *  _Duplication_
+ *
+ *  - #to_s:: If +self+ is a subclass of \String, returns +self+ copied into a \String;
+ *            otherwise, returns +self+.
+ *  - #to_str:: If +self+ is a subclass of \String, returns +self+ copied into a \String;
+ *              otherwise, returns +self+.
+ *
+ *  === Methods for Converting to Non-\String
+ *
+ *  Each of these methods converts the contents of +self+ to a non-\String.
+ *
+ *  <em>Characters and Bytes</em>
+ *
+ *  - #bytes:: Returns an array of the bytes in +self+.
+ *  - #chars:: Returns an array of the characters in +self+.
+ *  - #codepoints:: Returns an array of the integer ordinals in +self+.
+ *  - #getbyte:: Returns an integer byte as determined by a given index.
+ *
+ *  _Splitting_
+ *
+ *  - #lines:: Returns an array of the lines in +self+, as determined by a given record separator.
+ *  - #partition:: Returns a 3-element array determined by the first substring that matches
+ *                 a given substring or regexp,
+ *  - #rpartition:: Returns a 3-element array determined by the last substring that matches
+ *                  a given substring or regexp,
+ *  - #split, #to_a:: Returns an array of substrings determined by a given delimiter -- regexp or string --
+ *                    or, if a block given, passes those substrings to the block.
+ *
+ *  _Matching_
+ *
+ *  - #scan:: Returns an array of substrings matching a given regexp or string, or,
+ *            if a block given, passes each matching substring to the  block.
+ *  - #unpack:: Returns an array of substrings extracted from +self+ according to a given format.
+ *  - #unpack1:: Returns the first substring extracted from +self+ according to a given format.
+ *  - #grapheme_clusters:: Returns an array of the grapheme clusters in +self+.
```

Comment:
This is more similar to `bytes`, `chars`, and `codepoints`, I would move it into that subsection (you may want to rename that subsection).

Reply:
Fixed.